GoogleForCreators/web-stories-wp
# Added feature flag for Taxonomy support

## Context
New feature flag added to experiments for tags and categories.

## Summary
Add feature flag in order to turn on/off  the upcoming Taxonomy feature.

## Testing Instructions

While logged in as an admin, you should be able to see a new checkbox for Taxonomy under the Stories -> Experiments setting. 




This PR can be tested by following these steps:

1. Log in as an admin
2. Navigate to Stories -> Experiments
3. Look for the "Taxonomy" checkbox under "Editor"


## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---

Fixes # 8832
